spec/index.html

```diff
@@ -729,8 +730,9 @@ <h2>Literals</h2>
 
     <p>A literal is a <dfn>language-tagged string</dfn> if the third element
       is present and the fourth element is not present.
-      Lexical representations of language tags MAY be converted
-      to lower case.
+      Lexical representations of language tags
+      MUST be case normalized
+      and MAY (preferably) be converted to lower case.
```

Comment:
This is stronger than the requirement that there be one language tag for any case-insensitive sequence of characters. It will be covered by the text around 722.

Case normalization is an implementation technique. Another is a map with case insensitive keys (so first seen case choice of a langtag is used everywhere). System already exist that follow RFC5646 normalization and I do not believe RDF concepts should imply that are wrong in some way. The important factor is that data will interoperate.

Reply:
so, perhaps just remove the "and MAY (preferentially) be converted to lower case" part of this sentence?